src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -409,7 +409,7 @@
     "Get-AzCdnCustomDomain": "Get-AzureRmCdnCustomDomain",
     "New-AzCdnCustomDomain": "New-AzureRmCdnCustomDomain",
     "Remove-AzCdnCustomDomain": "Remove-AzureRmCdnCustomDomain",
-    "Enable-AzCdnCustomDomain": "Enable-AzureRmCdnCustomDomain",
+    "Enable-AzCdnCustomDomainHttps": "Enable-AzureRmCdnCustomDomainHttps",
```

Comment:
@hytao please revert the change made in this file -- it won't be necessary to update this file